### PR TITLE
docs: Added explanation to the Usage section of README

### PR DESCRIPTION
**Pull request by Aditya-Kumar-Sahu**
Added explanation to the Usage section of README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,9 +11,45 @@ The official starter template for ⛺ Tent.
 
 ## Usage
 
+### 1. Clone the starter project:
```
Comment (sebkolind) [inline README.md:14]: I also think we can omit the `1.` and `2.` etc.

**Comment by sebkolind** [comment]
@Aditya-Kumar-Sahu have you had the time to look into the comments I've given? It would be great to get this merged in 🙏🏻

**Comment by Aditya-Kumar-Sahu** [comment]
> @Aditya-Kumar-Sahu have you had the time to look into the comments I've given? It would be great to get this merged in 🙏🏻

Sorry, I am a little busy with my work. I will be free after 5th June. I can update it after that if it's fine with you.

**Comment by sebkolind** [comment]
> > @Aditya-Kumar-Sahu have you had the time to look into the comments I've given? It would be great to get this merged in 🙏🏻
> 
> Sorry, I am a little busy with my work. I will be free after 5th June. I can update it after that if it's fine with you.

That is perfect. Don't worry about it, take your time! ✌🏻

File: README.md [hunk 2]
```diff
@@ -11,9 +11,45 @@ The official starter template for ⛺ Tent.
...
+    └───src
+        ├───components
+        │   └───counter
+        ├───utils
+        └───__tests__
+```
+- **`.github/workflows`**: Contains GitHub Actions workflows for CI/CD.
+- **`src/components/counter`**: Holds the counter component code.
+- **`src/utils`**: Contains utility functions used throughout the project.
+- **`src/__tests__`**: Houses the test files.
```
[the remark this answers is not in the document]
Comment (Aditya-Kumar-Sahu) [reply README.md:34]: I think it is better the way it is, it might look a little mess with descriptions within tree.

**Comment by Aditya-Kumar-Sahu** [comment]
@sebkolind Do you need any more changes to be done? It is showing "1 change requested", but I'm unable to see any request.

**Comment by sebkolind** [comment]
> @sebkolind Do you need any more changes to be done? It is showing "1 change requested", but I'm unable to see any request.

I'm on vacation at the moment. Will get back to you next week👏🏻 Thanks for taking the time to contribute!

**Comment by Aditya-Kumar-Sahu** [comment]
> > @sebkolind Do you need any more changes to be done? It is showing "1 change requested", but I'm unable to see any request.
> 
> I'm on vacation at the moment. Will get back to you next week👏🏻 Thanks for taking the time to contribute!

Ok sure, enjoy 👍

**Review by sebkolind — APPROVED** [review]
@Aditya-Kumar-Sahu LGTM! Thank you for contributing. 🙏🏻

**Pull request merged**